examples/settings.py

```diff
@@ -95,6 +99,38 @@ def make_cfg(settings):
         ortho_sensor_spec.sensor_subtype = habitat_sim.SensorSubType.ORTHOGRAPHIC
         sensor_specs.append(ortho_sensor_spec)
 
+    # TODO Figure out how to implement copying of specs
+    def create_fisheye_spec(**kw_args):
+        fisheye_sensor_spec = habitat_sim.FisheyeSensorDoubleSphereSpec()
+        fisheye_sensor_spec.uuid = "fisheye_sensor"
+        fisheye_sensor_spec.sensor_type = habitat_sim.SensorType.COLOR
+        fisheye_sensor_spec.sensor_model_type = (
+            habitat_sim.FisheyeSensorModelType.DOUBLE_SPHERE
+        )
+        fisheye_sensor_spec.alpha = 0.59  # default
```

Comment:
I would add a comment here:
/*
The default value (alpha, xi) is set to match the lens "BF2M2020S23" found in Table 3 of this paper:
    Vladyslav Usenko, Nikolaus Demmel and Daniel Cremers: The Double Sphere
    Camera Model, The International Conference on 3D Vision (3DV), 2018

 You can find the intrinsic parameters for the other lenses in the same table as well.
*/